### ISSUE TITLE
Variable doesn't work in saved replies in the first attempt

### ISSUE THREAD

**Issue by goschka** [freescout-helpdesk/freescout#1038]
Hi,

I have installed the "saved replies" module and defined replies with variables. If I open a new message and select a saved reply, variables (like my user name) don't work in the first attempt. If I add the saved reply again or type anything else before placing the saved reply, the variables work correctly.

I have made a screencast where I show this issue: [https://jumpshare.com/v/8vaui5N0Pokwi1zPBtRu](https://jumpshare.com/v/8vaui5N0Pokwi1zPBtRu)

Any help is very appreciated :)

Thanks!
Best,
Georg

**Comment by freescout-helpdesk** [comment]
Unfortunately we can't reproduce the issue on our end.

**Comment by goschka** [comment]
Well, that's a pity. Any idea, what I can do about this issue? I definitly would like to use the module I paid for without the workaround I need to use now. Thanks!

**Comment by freescout-helpdesk** [comment]
Can you reproduce the issue on https://demo.freescout.net ?

**Comment by goschka** [comment]
No, unfortunatelly I can't. I even copied the complete setup (incl. database) to another server and activated only the "saved replies" module there with the same problem.
What I can reproduce on the demo server is the problem that when I try to add the variable into the saved reply it is not saved. I have to add it twice, then it saved once, see following screencast: [https://jumpshare.com/v/tsYsoeT2y1l3RVbhxA1Q](https://jumpshare.com/v/tsYsoeT2y1l3RVbhxA1Q)

Is it probably somehow related to my issue?

**Comment by freescout-helpdesk** [comment]
Unfortunately not.

**Comment by goschka** [comment]
I could provide you my complete setup including the database. Since I was able to reproduce the issue on an other server you might be able to reproduce it on yours?

**Comment by freescout-helpdesk** [comment]
SSH access to the server and access to the FreeScout may help (support@freescout.net)

**Comment by goschka** [comment]
Hi,

I have sent you an invitation for access to FreeScout on my test server to support@freescout.net. The access data for SSH is provided as a note in the single Ticket in "not assigned" folder. Since I have only one licence for the "saved reply" module and I need it active on my productive page: could you please activate the licence on the test server for the debug purposes with your own license?

Thanks a lot!

Best,
Georg

**Comment by freescout-helpdesk** [comment]
We've checked it in the TEST-Postfach mailbox - works fine. Which browser are you using?

**Comment by goschka** [comment]
I have tested two browsers:

- Chrome Version 88.0.4324.150: Just reproduced in this screencast: [https://jumpshare.com/v/oIycWDev2KTUVQzqYgZ5](https://jumpshare.com/v/oIycWDev2KTUVQzqYgZ5)
- Firefox 85.0.2: Just reproduced in this screencast: [https://jumpshare.com/v/qtmCQNLJviJIv7BWqyvS](https://jumpshare.com/v/qtmCQNLJviJIv7BWqyvS)

**Comment by goschka** [comment]
Hi, do you have any news on the issue? Thanks!

**Comment by freescout-helpdesk** [comment]
It looks like we can't help here.

**Comment by goschka** [comment]
But you obviously see the bug which I have shown in the screencasts [here](https://github.com/freescout-helpdesk/freescout/issues/1038#issuecomment-778648363), right? Were you able to reproduce it?

**Comment by freescout-helpdesk** [comment]
Nope.

**Comment by goschka** [comment]
So, when you click on the same page we are talking about on "create new message" and enter a quick reply right away, you cant't reproduce it? Could you make a screencast? This happens with two different browsers but not to you? Sorry, hard to believe.

**Comment by freescout-helpdesk** [comment]
Issue has not been reproduced on our end.

**Comment by goschka** [comment]
The error is reproducible on the server where I provided you (as you requested) the access to FreeScout and via SSH. I also offered you a copy of the setup, since the error is also reproducible after deployment to another server. 

Instead, you have decided to ignore the bug and to pull the “can’t reproduce” card. Very unprofessional. I know FreeScout is a free software and I can not expect much, but I have paid for the module which leads to the buggy issue. It would be fine if you have accepted that the bug is there and have admitted you don’t know the root cause right now. I’m a hardware developer myself and know that it can be frustrating to not be able to find the bug.

The way you decided to deal with the issue is just unprofessional and shows to me once again: you get what you pay for.

**Comment by afecte** [comment]
I have the same issue. Variables don't work until I do some action first. I tried here https://demo.freescout.net/ but it works fine.
I found the problem but not the solution. Problem is that there is a redirection from this url https://demo.freescout.net/mailbox/1/new-ticket to this url https://demo.freescout.net/conversation/34
But when I use my Freescout installation, this redirection only happens after editing some field. Any idea about how to solve it?
